packages/bundler/test/UserOpMethodHandler.test.ts

```diff
@@ -1,28 +1,21 @@
 import { BaseProvider, JsonRpcSigner } from '@ethersproject/providers'
-import { assert } from 'chai'
+import { assert, expect } from 'chai'
 import { ethers } from 'hardhat'
-
-import { ERC4337EthersProvider } from '@erc4337/client/dist/src/ERC4337EthersProvider'
-import { ERC4337EthersSigner } from '@erc4337/client/dist/src/ERC4337EthersSigner'
-import { SimpleWalletAPI } from '@erc4337/client/dist/src/SimpleWalletAPI'
-import { UserOpAPI } from '@erc4337/client/dist/src/UserOpAPI'
-import { UserOperation } from '@erc4337/common/src/UserOperation'
-
-// noinspection ES6UnusedImports
-import type {} from '@erc4337/common/src/types/hardhat'
+import { parseEther } from 'ethers/lib/utils'
 
 import { UserOpMethodHandler } from '../src/UserOpMethodHandler'
+
+import { BundlerConfig } from '../src/BundlerConfig'
+import { BundlerHelper, SampleRecipient } from '../src/types'
 import {
-  SimpleWallet,
   EntryPoint,
-  SampleRecipient,
-  SingletonFactory,
-  SimpleWallet__factory
-} from '@erc4337/common/src/types'
+  SimpleWalletDeployer__factory,
+  UserOperationStruct
+} from '@account-abstraction/contracts'
 
-import { BundlerConfig } from '../src/BundlerConfig'
-import { BundlerHelper } from '../src/types'
-import { ClientConfig } from '@erc4337/client/dist/src/ClientConfig'
+import 'source-map-support/register'
```

Comment:
I had very mixed success with 'source-map-support'. Is it working well for you? Or is it a placebo?

Reply:
at some cases it does. I didn't do a deep check where it works and where it isn't.